gulpfile.js

```diff
@@ -215,7 +220,18 @@ gulp.task('build-frontend-games-html', ['clean-frontend'], function () {
 
 gulp.task('build-frontend-js-vendor', ['clean-frontend'], function () {
   
-  return gulp.src(buildConfig.frontend.js.vendor.src)
+  var b = browserify({
+    entries: buildConfig.frontend.js.vendor.src,
+    debug: true
+  });
+
+  return b.bundle()
+    .pipe(source(buildConfig.frontend.js.vendor.src))
```

Comment:
It does, and solves a weird folder nesting issue in the vendor folder of build. I've commented out the uglify statement.